src/compiler/types.ts

```diff
@@ -10013,7 +10013,7 @@ export const enum ListFormat {
     SourceFileStatements = MultiLine | NoTrailingNewLine,
     Decorators = MultiLine | Optional | SpaceAfterList,
     TypeArguments = CommaDelimited | SpaceBetweenSiblings | SingleLine | AngleBrackets | Optional,

```

Comment:
Can this happen with type arguments within JSX too?

Reply:
No - it's a quirk of lambda parsing that they require a `,` , an `extends`, or a `=` clause to disambiguate them from a jsx tag.